src/InstagramClone.Api/Controllers/PostController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using InstagramClone.Api.Database;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InstagramClone.Api.Controllers
+{
+
+    [ApiController]
+    [Route("api/post")]
```

Comment:
I think this should be plural posts not post.

Reply:
ok